docs/_newsfragments/777.removal.rst

```diff
@@ -0,0 +1,4 @@
+The class :class:`~.falcon.http_error.NoRepresentation` was removed since
+all :class:`~.falcon.HTTPError` have optional representation.
+Deprecate the use of positional arguments for the optional kw args of
```

Comment:
Seems like this deprecation could be its own news fragment?

Reply:
I'm not sure how to make them. They should be 777 since tha is the issue, ans I think are both removal?

Reply:
You could make the other one "misc" since it hasn't been removed (as a breaking change) yet, *per se*. Also, I think it is reasonable to add a few (hyphenated) words of description after the issue number (as has been done with other fragments.

@csojinb @vytas7 thoughts? Related: https://github.com/falconry/falcon/pull/1621#discussion_r358016645

Reply:
I've used only the numbers as the names since I though that we wanted to use towncrier to automatically add the link to the issue for every changelog file.
I can make a new misc later

Reply:
Ahh, I see. I just tried that and indeed it drops a `(#{number})` reference in there, although it doesn't generate a GH link (perhaps a sphinx extension could be employed for this). That is nice, but on the other hand it seems like there should be a way to (1) have towncrier allow multiple entries for the same issue and (2) to better support items that don't have an associated issue, perhaps by simply not appending the `(name-of-news-fragment)`. Perhaps we need a post-processing script?

Reply:
There is a configuration parameter for towncrier to make it apply the gh link. We were trying it in gitter with @vytas7 

I can just change the name for now, and then the names of the newsitems will be changed after the configuration for towncrier is figured out

Reply:
the config was `issue_format = "https://github.com/falconry/falcon/issues/{issue}"`